website/common/locales/en/tasks.json

```diff
@@ -1,7 +1,7 @@
 {
     "clearCompleted": "Delete Completed",
     "lotOfToDos": "Your most recent 30 completed To-Dos are shown here. You can see older completed To-Dos from Data > Data Display Tool or Data > Export Data > User Data.",
-    "deleteToDosExplanation": "If you click the button below, all of your completed To-Dos and archived To-Dos will be permanently deleted. Export them first if you want to keep a record of them.",
+    "deleteToDosExplanation": "Permanently delete all of your completed and archived To-Dos. Except for To-Dos from active challenges and Groups Plans. Export them first if you want to keep a record of them.",
```

Comment:
Can you please change this so that the full stop after "To-Dos" is removed, and "Except" becomes "except". It's more grammatically correct to have "except..." as a part of the sentence. Please also change "Groups" to "Group" (singular not plural).

The completed line will then be:

"deleteToDosExplanation": "Permanently delete all of your completed and archived To-Dos except for To-Dos from active challenges and Group Plans. Export them first if you want to keep a record of them.",